lib/vectorsearch/pgvector.rb

```diff
@@ -73,7 +81,7 @@ def similarity_search_by_vector(embedding:, k: 4)
       result = client.transaction do |conn|
         conn.exec("SET LOCAL ivfflat.probes = 10;")
         query = <<~SQL
-          SELECT id, content FROM #{@index_name} ORDER BY vectors <-> $1 ASC LIMIT $2;
+          SELECT id, content FROM #{@index_name} ORDER BY vectors #{@operator} $1 ASC LIMIT $2;
```

Comment:
We should probably add:
```
attr_reader :operator
```
at the top of the class, so we're not accessing the instance variable directly.